### ISSUE TITLE
plans?

### ISSUE THREAD

**Issue by pkieltyka** [gameofbombs/gobi#1]
Hi @ivanpopelyshev very nice work porting pixi to TS along with adding some nice optimizations. I'm just wondering on the state of the project and if there are still plans to progress here or if attention/focus has shifted to pixi/next (v5) and just having a typings definition file?

thank you

**Comment by ivanpopelyshev** [comment]
Yep, I'm working on pixi-v5 `next` right now. I'm trying to port some optimizations from this repo, but people are not ready to review it yet https://github.com/pixijs/pixi.js/pull/4586

I certainly hasnt abandoned it, this project is used in development of one big game, but right now almost all the focus is flash-related, i want to be able to run SWFs :)

**Comment by ivanpopelyshev** [comment]
I dont know how to answer your question, I just enumerate my thourhgts i have about it.

There are also new optimizations for spine masks in https://github.com/gameofbombs/pixi-heaven/  , it works for v4. 

Some ideas from this fork are already present in v5. 

The idea of TS fork is that 

1. it doesnt have dependencies, everything is typed. 

2. it contains some experimental optimizations that i like which are not possible to merge in vanilla pixi

**Comment by pkieltyka** [comment]
This sounds awesome and I might try using gobi directly for my next game then.
